apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,25 +154,14 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
-      strategy.hasDeterminativeProgress !== undefined &&
-      strategy.hasDeterminativeProgress();
-  };
-
-  //
   // Shows the progress dialog based on the
   //
   var _displayProgress = function _displayProgress() {
-    var progressClass = determinativeProgress ? 'progressBar' : 'spinner';
 
     Contacts.utility('Overlay', function _loaded() {
       progress = utils.overlay.show(
         strategy.getExportTitle(),
-        progressClass,
+        true,
```

Comment:
ok, but it's still wrong to use Boolean value instead of String. You calling function ``utils.overlay.show()`` here, it have following signature: ``function show(messageId, progressClass, textId)`` and according to source code expected values for ``progressClass`` parameter are: ``'spinner', 'activityBar', 'progressActivity', 'progressBar'``. So in case if you are not going to refactor shared component which implements this overlay, you shall pass ``'progressBar'`` value.